src/ThreeEditor/util/Beam.ts

```diff
@@ -23,6 +24,7 @@ const _default = {
 	position: new THREE.Vector3(0, 0, 0),
 	direction: new THREE.Vector3(0, 0, 1),
 	energy: 150,
+	energySpread: 250,
```

Comment:
good default value for energy spread is 1% of energy mean, here it should be 1.5